discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -160,6 +160,23 @@ def set_ssh_key(self, ssh_key):
     def set_base_dns_domain(self, base_dns_domain):
         logging.info(f"Setting base DNS domain:{base_dns_domain} for cluster: {self.id}")
         self.api_client.update_cluster(self.id, {"base_dns_domain": base_dns_domain})
+    
+    def set_advanced_networking(self, cluster_cidr, service_cidr, cluster_host_prefix):
```

Comment:
I think just this `set_advanced_networking` is enough no? Have defaults for the params and build the dict from them. Or just except **kwargs
wdty?

Reply:
The only issue is if you only want to update one of the parameters you will have to update all three with defaults.

Reply:
/lgtm